schema.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TABLE genre (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE author (
+     id SERIAL PRIMARY KEY,
+     first_name VARCHAR(255),
+     last_name VARCHAR(255)
+);
+
+CREATE TABLE source (
+     id SERIAL PRIMARY KEY,
+     name VARCHAR(255)
+);
+
+CREATE TABLE label (
```

Comment:
# Hi @mosidrum ,

### You've done a great job! 

### I think schema related to book and label class is assigned to your coding partner @anita00001 (Please check [issue 9](https://github.com/anita00001/Catalog-of-my-things/issues/9))
You may want to remove the table label.